### PR TITLE
Remove mentions of hardwoods

### PR DESCRIPTION
**Pull request by almasaeed2010**
This is a simple change to remove all mentions of the old `hardwoods_jbrowse` module.

See #14 and #13

### CODE REVIEW

File: tripal_jbrowse_mgmt/tripal_jbrowse_mgmt.info [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name = Hardwoods Jbrowse
-description = Create JBrowse instances.
+name = Tripal Jbrowse Management
```
Comment (laceysanderson) [inline tripal_jbrowse_mgmt/tripal_jbrowse_mgmt.info]:
```suggestion
name = Tripal JBrowse Management
```

File: tripal_jbrowse_mgmt/tripal_jbrowse_mgmt.module [hunk 2]
```diff
@@ -108,11 +108,11 @@ function tripal_jbrowse_mgmt_menu() {
 function tripal_jbrowse_mgmt_permission() {
   $items = [];
 
-  $items['configure hardwoods jbrowse'] = [
-    'title' => t('Configure Hardwoods JBrowse'),
+  $items['configure tripal_jbrowse_mgmt'] = [
+    'title' => t('configure tripal_jbrowse_mgmt'),
```
Comment (laceysanderson) [inline tripal_jbrowse_mgmt/tripal_jbrowse_mgmt.module]:
```suggestion
    'title' => t('Configure Tripal JBrowse Management'),
```

**Review by laceysanderson — APPROVED** [review]
Overall, looks good 👍 I tested on my Development site and all seems to still work as expected.

**Comment by almasaeed2010** [comment]
Thanks for testing @laceysanderson

**Pull request merged**